src/utility/index.jsx

```diff
@@ -96,22 +95,7 @@ export function fromNow(input, tooltip = false) {
   }
 
   if (tooltip) {
-    const tooltipId = uuid.v4();
-
-    return (
-      <div data-tip data-for={tooltipId}>
-        {fromNow}
-        <ReactTooltip id={tooltipId} place="right" type="light" effect="float">
-          { // Country Code Language List http://www.fincher.org/Utilities/CountryLanguageList.shtml
-            date.toLocaleDateString('en-US', {
-              day: 'numeric',
-              month: 'short',
-              year: 'numeric',
-            })
-          }
-        </ReactTooltip>
-      </div>
-    );
+    return <FromNowTooltip fromNow={fromNow} date={date} />;
```

Comment:
I was actually thinking:
* The fromNow(date) function should convert a date to a string.  This should a standalone utility function.
* The FromNowTooltip component should accept a date, and then have a reference to the fromNow() function in utility, which it uses to build the markup.